pkg/kubelet/dockertools/manager.go

```diff
@@ -216,12 +218,30 @@ func NewDockerManager(
 		cpuCFSQuota:            cpuCFSQuota,
 	}
 	dm.runner = lifecycle.NewHandlerRunner(httpClient, dm, dm)
-	if serializeImagePulls {
-		dm.imagePuller = kubecontainer.NewSerializedImagePuller(kubecontainer.FilterEventRecorder(recorder), dm, imageBackOff)
+	dm.containerGC = NewContainerGC(client, containerLogsDir)
+
+	minimumDockerAPIVersionForParallelPullsFlag := false
+	version, err := dm.Version()
+	if err == nil {
+		result, err := version.Compare(minimumDockerAPIVersionForParallelPulls)
+		if err == nil && result >= 0 {
+			minimumDockerAPIVersionForParallelPullsFlag = true
+		}
+	}
+
+	if !minimumDockerAPIVersionForParallelPullsFlag {
```

Comment:
I think it's easier to reason through positive cases - can you switch this check?